### ISSUE TITLE
Broken image in weekly notification mail

### ISSUE THREAD

**Issue by noamoss** [OpenBudget/BudgetKey#506]
![Screenshot_2020-10-17-10-24-02-102_com.google.android.gm.jpg](https://user-images.githubusercontent.com/3581741/96331059-ec2acf00-1062-11eb-821d-f26e9197ac7b.jpg)

**Comment by noamoss** [comment]
One of the broken images links:
https://next.obudget.org/i/tenders/office/653603/15/9.2020

**Comment by akariv** [comment]
Can you forward me the email?

**Comment by noamoss** [comment]
@akariv while looking at it again, the problem did not reoccur, so I guess no point in forwarding it. Probably a slow connection with timeout or a temporary glitch.

Anyhow, I would suggest using [relevant techniques](https://exceptionshub.com/how-to-hide-image-broken-icon-using-only-csshtml-without-js.html), as a textual placeholder ("פריט לתצוגה") for these rare cases. 

(please notice I had 5 broken images icons in the mail at a first glance). 

What do you think?

**Comment by akariv** [comment]
Yes, adding an alt attribute is a good idea, also for accessibility reasons.
